From e3976558d4fa0a4a37826e8d1f43d7e578c7dce5 Mon Sep 17 00:00:00 2001
From: fengmushu <fengmushu@gmail.com>
Date: Sat, 11 Aug 2018 14:34:55 +0800
Subject: [PATCH] fixup kernel version compile

---
 natcap_common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/natcap_common.h b/natcap_common.h
index e7f36e0..2f7ea5d 100644
--- a/natcap_common.h
+++ b/natcap_common.h
@@ -231,7 +231,7 @@ static inline unsigned int optlen(const u_int8_t *opt, unsigned int offset)
 static inline u_int32_t tcpmss_reverse_mtu(struct net *net, const struct sk_buff *skb)
 {
 	struct flowi fl;
-#if ! defined(NF_MOVED_ROUTE_INDIRECTION) && LINUX_VERSION_CODE < KERNEL_VERSION(4, 16, 0)
+#if ! defined(NF_MOVED_ROUTE_INDIRECTION) && LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 0)
 	const struct nf_afinfo *ai;
 #endif
 	struct rtable *rt = NULL;
@@ -242,7 +242,7 @@ static inline u_int32_t tcpmss_reverse_mtu(struct net *net, const struct sk_buff
 	fl4->daddr = ip_hdr(skb)->saddr;
 
 	rcu_read_lock();
-#if ! defined(NF_MOVED_ROUTE_INDIRECTION) && LINUX_VERSION_CODE < KERNEL_VERSION(4, 16, 0)
+#if ! defined(NF_MOVED_ROUTE_INDIRECTION) && LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 0)
 	ai = nf_get_afinfo(PF_INET);
 	if (ai != NULL)
 		ai->route(net, (struct dst_entry **)&rt, &fl, false);
-- 
2.17.1

